sbysrc/sby.py

```diff
@@ -341,7 +325,9 @@ def run_job(taskname):
             backup_idx = 0
             while os.path.exists("{}.bak{:03d}".format(my_workdir, backup_idx)):
                 backup_idx += 1
-            early_log(my_workdir, "Moving directory '{}' to '{}'.".format(my_workdir, "{}.bak{:03d}".format(my_workdir, backup_idx)))
+            early_log(
+                my_workdir, "Moving directory '{}' to '{}'.".format(my_workdir,
+                                                                    "{}.bak{:03d}".format(my_workdir, backup_idx)))
```

Comment:
Some of these ways how it breaks long lines are really awkward. Are you sure this is an improvement?

Reply:
break is on 120 chars per line (same as in nextpnr c++ code), split of line have lot of configuration possibilities  (check .style.yapf ) but agree some are even looking strange a standard way of formatting python.

Reply:
120 chars per line seems fine. I've tried playing around with the options but none of them seem to change anything for this line?